pymodbus/client/sync_diag.py

```diff
@@ -1,4 +1,7 @@
 """Sync diag."""
+
+__all__ = ["ModbusTcpDiagClient", "get_client"]
```

Comment:
Cleanup formatting please.

Reply:
What do you want cleaned up here?

Reply:
the entries should be one on each line, that is the standard we have been following in all new files.

Last time I checked black did not put that back as one line.